README.md

```diff
@@ -31,6 +34,54 @@ func doUpdate(url string) error {
 }
 ```
 
+## Managed update
+
+To help make self updating Fyne application a new API and a tool, `selfupdatectl` have been introduced. The new API allow to provide a source where to get an update, configure a schedule for the update and a ed25519 public key to ensure that the update is only applied if they come from the proper source.
+
+Example with the new API:
+
+```go
+import (
+	"crypto/ed25519"
+	"log"
+	"time"
+
+	"github.com/fynelabs/selfupdate"
+)
+
+func main() {
+	done := make(chan struct{}, 2)
+
+	// Used `selfupdatectl create-keys` followed by `selfupdatectl print-key`
+	publicKey := ed25519.PublicKey{178, 103, 83, 57, 61, 138, 18, 249, 244, 80, 163, 162, 24, 251, 190, 241, 11, 168, 179, 41, 245, 27, 166, 70, 220, 254, 118, 169, 101, 26, 199, 129}
+
+	// The public key above match the signature of the below file served by our CDN
+	httpSource := selfupdate.NewHTTPSource(nil, "http://geoffrey-test-artefacts.fynelabs.com/nomad.exe")
```

Comment:
I think this should not be an OS specific string - would it be OK to use the template?